challenge_5/python/ekstrah/README.md

```diff
@@ -0,0 +1,3 @@
+Challenge 5 with Python 3.x
+
+The problem with my code here is that the Test Case Number 4 doesn't work since the input value is "" & "". (I don't think test case number 4 doesn't fit with permise)
```

Comment:
After rereading the premise of challenge #5 I believe you are correct in that test_4 doesn't  exactly fit the premise but this is simply an edge case that you can test for and return the expected output.